website/docs/cli/configuration/configuration.mdx

```diff
@@ -24,15 +24,58 @@ Think of this file as where you [bootstrap the settings or configuration of your
 [terraform](/core-concepts/components/terraform), then [this is where](/cli/configuration/components#terraform-component-behavior)
 you'd specify the command to run (e.g. [`opentofu`](/core-concepts/projects/configuration/opentofu)),
 the base path location of the components, and so forth.
+## Base Path
 
-## Configuration File (`atmos.yaml`)
+The base path defines the root directory for components, stacks, workflows, and Atmos configurations. It can be set using the `ATMOS_BASE_PATH` environment variable or by passing the `--base-path` command-line argument. If both are set, the `--base-path` will take precedence. Both absolute and relative paths are supported. When a relative path is provided, it will be expanded to an absolute path relative to the current working directory, as Atmos executes commands that may change the working directory (e.g., Terraform).
```

Comment:
```suggestion
The base path defines the root directory for components, stacks, workflows, and Atmos configurations. It can be set using the `ATMOS_BASE_PATH` environment variable or by passing the `--base-path` command-line argument. If both are set, the `--base-path` will take precedence. Both absolute and relative paths are supported. When a relative path is provided, it will be expanded to an absolute path relative to the current working directory. This is because Atmos executes commands that may change the working directory (e.g., Terraform).
```

Reply:
It mostly affects the `terraform-provider-utils` which reads the atmos configs. 

In geodesic we are usually setting the `ATMOS_BASE_PATH` for users.

Reply:
Any relative path is not compatible with the `terraform-provider-utils`, because terraform changes the directory.

Reply:
We hope to remove the for the utils provider in the future versions of our components and refarch.